src/server/debug.rs

```diff
@@ -228,48 +228,27 @@ impl Debugger {
         Ok(())
     }
 
-    /// Set a region to tombstone by manual, and apply other status(such as
+    /// Set regions to tombstone by manual, and apply other status(such as
     /// peers, version, and key range) from `region` which comes from PD normally.
-    pub fn set_region_tombstone(&self, id: u64, region: Region) -> Result<()> {
+    pub fn set_region_tombstone(&self, regions: Vec<Region>) -> Result<Vec<(u64, Error)>> {
+        let store_id = self.get_store_id()?;
         let db = &self.engines.kv_engine;
-        let key = keys::region_state_key(id);
+        let wb = WriteBatch::new();
 
-        let old_region = box_try!(db.get_msg_cf::<RegionLocalState>(CF_RAFT, &key))
-            .ok_or_else(|| Error::Other("Not a valid region".into()))
-            .map(|mut region_local_state| region_local_state.take_region())?;
+        let mut errors = Vec::with_capacity(regions.len());
+        for region in regions {
+            let region_id = region.get_id();
+            if let Err(e) = set_region_tombstone(db.as_ref(), store_id, region, &wb) {
+                errors.push((region_id, e));
```

Comment:
if we meet error, do we need to return directly?

Reply:
I think collect all errors for all regions is better. Just like if we run
```bash
find . -name *.rs -exec ls {} \;
```
if `find` meets permission denyed for some sub dirs, it won't exit immediately but keep listing others.